contrib/appengine-standard-util/src/main/java/io/opencensus/contrib/appengine/standard/util/AppEngineCloudTraceContextUtil.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.appengine.standard.util;
+
+import com.google.apphosting.api.CloudTraceContext;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.SpanId;
+import io.opencensus.trace.TraceId;
+import io.opencensus.trace.TraceOptions;
+import java.nio.ByteBuffer;
+
+/**
+ * Utility class to convert between {@link io.opencensus.trace.SpanContext} and {@link
+ * CloudTraceContext}.
+ *
+ * @since 0.14
+ */
+public final class AppEngineCloudTraceContextUtil {
+  private static final byte[] INVALID_TRACE_ID =
+      TraceIdProto.newBuilder().setHi(0).setLo(0).build().toByteArray();
+  private static final long INVALID_SPAN_ID = 0L;
+  private static final long INVALID_TRACE_MASK = 0L;
+
+  /**
+   * Converts AppEngine {@code CloudTraceContext} to {@code SpanContext}.
+   *
+   * @param cloudTraceContext the AppEngine {@code CloudTraceContext}.
+   * @return the converted {@code SpanContext}.
+   */
+  public static SpanContext toSpanContext(CloudTraceContext cloudTraceContext) {
```

Comment:
Since this is already an OpenCensus library, I think it would be clearer to call the methods toCloudTraceContext and fromCloudTraceContext.  Those names are also more consistent with the method names in StatusConverter, i.e., to/fromGrpcCode and to/fromGrpcStatus.

Reply:
done.